django_project/frontend/utils/data_table.py

```diff
@@ -215,10 +244,14 @@ def activity_report(queryset: QuerySet, request) -> Dict[str, List[Dict]]:
         end_year = request.GET.get("end_year")
         filters["year__range"] = (start_year, end_year)
 
-    activity_types = [
-        "Planned_euthanasia", "Planned_hunt/cull", "Planned_translocation",
-        "Unplanned/natural_deaths", "Unplanned/illegal_hunt"
-    ]
+    activity = request.GET.get("activity")
+    if activity:
+        activity_types = [activity]
+    else:
+        activity_types = [
+            "Planned_euthanasia", "Planned_hunt/cull", "Planned_translocation",
```

Comment:
Please do not change the names of the activity_type, it'll cause confusion and extra checks

Reply:
Okay @dimasciput